supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -21,3 +21,72 @@ When the format _day Month year_ causes a presentation or clarity issue, use ins
 
 [[product-names-versions-ref]]
 == Product names and version references
+
+////
+PROPOSED ADDITIONS?
+master-docinfo.xml specific and portal-specific version variables
+URL fragments and composed links?
+correct sequence when combining attributes
+ensure proper sequence for compound attributes
+syntax recommendations: what separators to use? dashes/hyphens, underscores?
+////
+
+Whenever you refer to the name of your product in full, or in its abbreviated form, or when you refer to the major and minor version of your product, avoid using hard-coded references and use attributes instead.
+
+Only use hard-coded version references if the version that you are referring to in a particular case never changes.  
+
+=== Attribute file
+
+Define attributes for product name and product version in a single common shared attributes file for each documentation set or product.
+
+Name the file `attributes.adoc`.
+You may use a different name if necessary, but if you do, ensure that the name of the file indicates to your collaborators that it is an attribute file.
+
+//For a single title, store the file in the same directory as the `master.adoc` file.
+For sharing between multiple titles, store the file in a common directory above the directories that contain your titles, such as `docs/common/` or `docs/topics/templates`.
+
+Include the attributes file at the beginning of the `master.adoc` files of all titles in your documentation set:
+
+.master.adoc
+[source,asciidoc,options="nowrap"]
+----
+\include::common/attributes.adoc[]
+----
+
+TIP: Create a symbolic link pointing to the `common` directory inside the directory containing the `master.adoc` file of your title. It makes including the attribute file in your title easier. +
+Do not copy the directory or attribute file directly, and do not use hard links!
+
+=== Minimum required attributes
+
+For each title ensure that you define attributes for the following values:
+
+* product name
+* abbreviated product name (if applicable to your product)
+* the major and minor version of the product (for example, 5 and 5.4)
+
+You may create additional attributes according to what your documentation requires.
+For example, some projects with complex names may define multiple abbreviated and non-abbreviated forms for their product name. You may also combine existing product name attributes to create compound product names.
+
+=== Attribute usage
+
+Product names and abbreviations::
+Use the product name attribute for all instances of the product name where possible.
+Avoid using hard-coded product names.
+
+Product versions::
+Use the product version attribute for instances where the product version should change with the next version and the content would still be correct.
+Do not use the product version attribute if the version should not change.
+For example, if a feature was introduced in a certain version, the version should be hard-coded.
+
+////
```

Comment:
I realize that it's commented out, but my vote would be to drop the topic **Recommendations on attribute name syntax**. At least for now.

Reply:
@gregraka I removed the section and incorporated parts of the information from it into the list iunder _Minimum required attributes_